### ISSUE TITLE
Log: Wox got an error

### ISSUE THREAD

**Issue by meanmechanics** [Wox-launcher/Wox#692]
Wox version: 1.3.67
OS Version: Microsoft Windows NT 10.0.10586.0
Date: 05/25/2016 09:07:03
Exception:

System.Runtime.InteropServices.COMException
The component cannot be found. (Exception from HRESULT: 0x88982F50)
## English

Before you report any bugs try search existing issues
1. Please specify Wox version
   - If you checkout from master/dev branch, please specify **commit number**
2. Please describe the steps to **reproduce** the bug
3. Please provide Windows version
4. Please provide Exception info if you have.
5. Please provide **logs** from `%APPDATA%\Wox\Logs` if you have.
6. Please provide screenshot if you have
## 中文 / Chinese

报 bug 之前搜索一下已经有的 issue
1. 请提供 Wox 版本
   - 如果你是从 master/dev branch checkout 的，请提供 **commit number**
2. 请提供如何**重现** bug 的步骤
3. 请提供 Windows 版本
4. 如果有 Exception/Crash Report 的话，请一并提供
5. 如果有 **logs** （`%APPDATA%\Wox\Logs`）的话，请一并提供
6. 如果有截图的话，请一并提供

**Comment by bao-qian** [comment]
Please provide logs from %APPDATA%\Wox\Logs

**Comment by meanmechanics** [comment]
2016-05-24 12:10:42> Program: Starting Squirrel Updater: --processStart Wox.exe
2016-05-24 12:10:42> Program: Want to launch 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe'
2016-05-24 12:10:42> Program: About to launch: 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe': 
2016-05-24 12:42:32> Program: Starting Squirrel Updater: --processStart Wox.exe
2016-05-24 12:42:32> Program: Want to launch 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe'
2016-05-24 12:42:32> Program: About to launch: 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe': 
2016-05-25 09:02:04> Program: Starting Squirrel Updater: --processStart Wox.exe
2016-05-25 09:02:04> Program: Want to launch 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe'
2016-05-25 09:02:04> Program: About to launch: 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe': 
2016-05-25 09:16:56> Program: Starting Squirrel Updater: --processStart Wox.exe
2016-05-25 09:16:56> Program: Want to launch 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe'
2016-05-25 09:16:56> Program: About to launch: 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe': 
2016-05-25 09:17:15> Program: Starting Squirrel Updater: --processStart Wox.exe
2016-05-25 09:17:16> Program: Want to launch 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe'
2016-05-25 09:17:16> Program: About to launch: 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe': 
2016-05-25 16:57:05> Program: Starting Squirrel Updater: --processStart Wox.exe
2016-05-25 16:57:05> Program: Want to launch 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe'
2016-05-25 16:57:05> Program: About to launch: 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe': 
2016-05-25 17:00:05> Program: Starting Squirrel Updater: --processStart Wox.exe
2016-05-25 17:00:05> Program: Want to launch 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe'
2016-05-25 17:00:05> Program: About to launch: 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe': 
2016-05-25 17:00:19> Program: Starting Squirrel Updater: --processStart Wox.exe
2016-05-25 17:00:19> Program: Want to launch 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe'
2016-05-25 17:00:19> Program: About to launch: 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe': 
2016-05-25 17:00:19> Program: Starting Squirrel Updater: --processStart Wox.exe
2016-05-25 17:00:19> Program: Want to launch 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe'
2016-05-25 17:00:19> Program: About to launch: 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe': 
2016-05-25 17:00:34> Program: Starting Squirrel Updater: --processStart Wox.exe
2016-05-25 17:00:34> Program: Want to launch 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe'
2016-05-25 17:00:34> Program: About to launch: 'C:\Users\Alex\AppData\Local\Wox\app-1.3.67\Wox.exe':

**Comment by bao-qian** [comment]
Please provide logs from %APPDATA%\Wox\Logs

**Comment by meanmechanics** [comment]
apologies did not pay attention to your query
[wox-logs-2016-05-24.zip](https://github.com/Wox-launcher/Wox/files/282814/wox-logs-2016-05-24.zip)

**Comment by bao-qian** [comment]
you add `C:\tools` in wox programs plugin, but you didn't set correct directory permission for `C:\tools`. as a result, wox can't read it.
if you don't know how to set windows directory correctly, I'd suggest you put `C:\tools` in your user directory. e.g. %USERPROFILE%

**Comment by meanmechanics** [comment]
Hello happlebao .
I am a total noob so please excuse my ignorance though I got some questions and comments. 
I could learn to set "correct directory permission" or "set windows directory"
My apologies if I seem not to understand you and if I take your time a little more:
1. what directory permission? In my ignorance I put all "portable" or "standalone" softwares in C:\Tools so as current user have write permission...
2. I really don't intend to put my Tools directory in %USERPROFILE% because of the previous comment and because I am not ready to give up on my other softwares or ease I get by using them from the C:\Tools. Is there an easy way, for the newb that I am, to continue to do so while using wox? All that I need is that Wox index the TOOLS directory so I can launch the .exe, .lnk or .bat etc.

If this is going to be complicated forget it, it is not really important. Someone told me how much better Wox was over Launchy so I decided to give it a try. At first I was not going to post the log : "wox got an error" but I thought that may be the dev may find interesting (if not, why the prompt window message telling me to post it here?) how I am using his software and how he can improve his software.

Thanks anyway

**Comment by bao-qian** [comment]
I think wox needs at least following permission
1. read & execute
2. list folder contents
3. read

post your properties tab to here like this one
![image](https://cloud.githubusercontent.com/assets/1169912/15559085/1e438d48-22d7-11e6-97c2-0035881fa6d6.png)

and screenshot of task manager
![image](https://cloud.githubusercontent.com/assets/1169912/15559111/45415a06-22d7-11e6-8edb-9562515b6d63.png)

try grant more permission until there are not more errors in logs like this
`Access to the path xxx is denied.`

**Comment by meanmechanics** [comment]
hello. pls find attached. [folder_list-PROPSecurityTab-Task_wox.zip](https://github.com/Wox-launcher/Wox/files/285088/folder_list-PROPSecurityTab-Task_wox.zip)

**Comment by bao-qian** [comment]
1. `File list.text` is not a plain text, and I don't know what you are trying to include in this file.
2. 
   ![image](https://cloud.githubusercontent.com/assets/1169912/15586433/2dfe4188-237c-11e6-81aa-ebccfad708e9.png)
   ![image](https://cloud.githubusercontent.com/assets/1169912/15586437/34dfba7c-237c-11e6-8d44-b1318bdbf70c.png)

as you can see, the user is not correct in property tab

**Comment by meanmechanics** [comment]
Dear happlebao,
1. file list.txt is a text file generated by a neat little software call SpaceSniffer [http://www.uderzo.it/main_products/space_sniffer/]. What I am trying to include in this file?? What you asked "2. List folder contents..." please see included screenshot for a visual explanation.
2. Please see included screenshot.
![screen-2016-05-27_22-59-12](https://cloud.githubusercontent.com/assets/18630177/15620124/68ce1a48-2469-11e6-9229-1b4b9b9141b1.jpg)
![screen-2016-05-27_22-53-26](https://cloud.githubusercontent.com/assets/18630177/15620141/8130493a-2469-11e6-9061-175fe84fee6f.jpg)

**Comment by bao-qian** [comment]
1. "List folder contents" is one of the _permission_ wox needs. I'm not asking you to include     the whole directory tree
2. wait a moment, I'm asking you to upload the security tab of `C:\Tools` properties.
3. use Task Manager, not process hacker. So it can display the whole username of running process. and upload screenshot again.

**Comment by meanmechanics** [comment]
Dear Happlebao,

ok, ok. I think we did not start well. I may be a bit stressed by work and not really reading what you are telling me. Anyway I already broke wox :D and got some more questions. In a couple of days I'm going to to uninstall wox and start fresh. I'm also going to install on another computer with a "classic structure" and on a pen drive. If everything is ok I'm going to post here if not I'm going to post in the wox discussion. That said wox is sleek,your patience is gold though I'm having a rough time understanding you. tks a lot
